scripts/common/sonic-utilities-build/build.sh

```diff
@@ -31,7 +31,10 @@ sudo dpkg -i buildimage/target/debs/buster/python2-yang_1.0.73_amd64.deb
 
 cd sonic-utilities
 
-# Test building the Python wheel
+# Run unit tests
+sudo python setup.py test
```

Comment:
the problem is that test report is missing. will this put the report in the same location?

Reply:
It will put it in a different location, which I already changed in https://github.com/Azure/sonic-build-tools/pull/160